config/localConfig.template.php

```diff
@@ -34,6 +34,9 @@
 /* Google Analytics Tracking Code */
 define('GA_TRACKING_CODE', '');
 
+/* Course Language */
+define('COURSE_LANGUAGE', '');
```

Comment:
For consistency, please make the constant the same name as the environment variable in Heroku.